make/test/BuildMicrobenchmark.gmk

```diff
@@ -93,6 +94,7 @@ $(eval $(call SetupJavaCompilation, BUILD_JDK_MICROBENCHMARK, \
     DISABLED_WARNINGS := processing rawtypes cast serial, \
     SRC := $(MICROBENCHMARK_SRC), \
     BIN := $(MICROBENCHMARK_CLASSES), \
+    JAVAC_FLAGS := $(JAVAC_FLAGS) --add-exports java.base/sun.security.util=ALL-UNNAMED, \
```

Comment:
Why do you need to add $(JAVAC_FLAGS) here?

Reply:
I'm trying to benchmark a class that is in a non-exported package `sun.security.util`. Without this line the benchmark doesn't compile. I couldn't find any other benchmarks that access non-exported classes, so I came up with my own implementation. 

Is there a better way to get the benchmark to compile?

Reply:
Adding "--add-exports java.base/sun.security.util=ALL-UNNAMED" is fine, I'm asking about "$(JAVAC_FLAGS)".

Reply:
Hm, maybe you just misunderstand how this makefile construct works. If you just want to add "--add-exports java.base/sun.security.util=ALL-UNNAMED", then that's all you should put in this assignment.

Reply:
yeah, I'm new to makefiles. Let me try that...

Reply:
Removed. I could have sworn I tried this... but apparently I didn't. Thanks for the suggestion!